libs/sdk-ui-kit/styles/scss/header.scss

```diff
@@ -612,21 +612,70 @@ $button-normal-active-shadow: transparentize($button-normal-active-border-color,
     display: flex;
     flex: 1 0 auto;
 
+    @mixin icon-state {
+        &.disabled {
+            color: $gd-color-disabled;
+        }
+
+        &.is-normal {
+            color: $gd-color-link;
+        }
+    }
+
+    @mixin hover-effect {
+        &:hover {
+            z-index: 100;
+            box-shadow: 0 0 0 1px $gd-color-highlight;
+
+            @include icon-state;
+        }
+    }
+
     .icon-sources {
         border-bottom-right-radius: 0;
         border-top-right-radius: 0;
 
-        &:hover {
-            border-color: $gd-color-highlight;
+        &.is-normal::before {
+            margin-top: 2px;
+            margin-left: 2px;
```

Comment:
Question: why normal state have margin, but active and disabled state are not?

Reply:
Thanks for this. I applied it to all state of model